main/sampleextension.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
We are implementing our own extension so we do not need/want the example implementation.